src/ngx_http_lua_headers.c

```diff
@@ -145,10 +155,27 @@ ngx_http_lua_ngx_req_raw_header(lua_State *L)
 
     dd("size: %d", (int) size);
 
+#if defined(nginx_version) && nginx_version >= 1011011
+    busy_bufs = ngx_array_create(r->pool, hc->nbusy + 1, sizeof(ngx_buf_t *));
```

Comment:
`ngx_array_create` may return `NULL`. We should check that here.

I'm a bit worried about allocating inside the request memory pool since it will lead to "memory leaks" when the user Lua code calls this Lua API function in a tight loop. Can we allocate it elsewhere?